Shared/Tests.Shared/PropertyChangedTests.cs

```diff
@@ -36,6 +39,8 @@ public class PropertyChangedTests
 
         private Realm _realm => _lazyRealm.Value;
 
+        private static readonly Task _completedTask = Task.FromResult<object>(null);
```

Comment:
I could always up the target framework of the tests assembly to get `Task.CompleteTask` back...